crates/uniswapx-rs/src/order.rs

```diff
@@ -82,21 +83,69 @@ sol! {
         PriorityCosignerData cosignerData;
         bytes cosignature;
     }
+
+    #[derive(Debug)]
+    struct V3DutchOrder {
+        OrderInfo info;
+        address cosigner;
+        uint256 startingBaseFee;
+        V3DutchInput baseInput;
+        V3DutchOutput[] baseOutputs;
+        V3CosignerData cosignerData;
+        bytes cosignature;
+    }
+
+    #[derive(Debug)]
+    struct V3CosignerData {
+        uint256 decayStartBlock;
+        address exclusiveFiller;
+        uint256 exclusivityOverrideBps;
+        uint256 inputAmount;
+        uint256[] outputAmounts;
+    }
+
+    #[derive(Debug)]
+    struct NonlinearDutchDecay {
+        uint256 relativeBlocks;
+        int256[] relativeAmounts;
+    }
+
+    #[derive(Debug)]
+    struct V3DutchInput {
+        address token;
+        uint256 startAmount;
+        NonlinearDutchDecay curve;
+        uint256 maxAmount;
+        uint256 adjustmentPerGweiBaseFee;
+    }
+    
+    #[derive(Debug)]
+    struct V3DutchOutput {
+        address token;
+        uint256 startAmount;
+        NonlinearDutchDecay curve;
+        address recipient;
+        uint256 minAmount;
+        uint256 adjustmentPerGweiBaseFee;
+    }
 }
 
 pub const MPS: u64 = 1e7 as u64;
+pub const BPS: Uint<256, 4> = Uint::from_limbs([10000, 0, 0, 0]);
```

Comment:
why can't we just do u64 like above?

Reply:
The thing it's being added to (exclusivityOverrideBps) is of type `Uint<256>`